lib/tests/tests/functions/parse_etld/custom_psl_file.vrl

```diff
@@ -0,0 +1,5 @@
+# object: { "host": "vector.customdev" }
+# result: { "etld": "customdev", "etld_plus": "vector.customdev", "known_suffix": true }
+
+etld_result = parse_etld!(.host, plus_parts: 1, psl: "lib/tests/tests/functions/custom_public_suffix_list.dat")
+etld_result
```

Comment:
nitpick 🥜 , optional suggestion

```suggestion
parse_etld!(.host, plus_parts: 1, psl: "lib/tests/tests/functions/custom_public_suffix_list.dat")
```

Reply:
Makes sense, I guess I copied it over from some other test and didn't properly clean up afterwards :smile: